react-native/react/native/listen-log-ui.js

```diff
@@ -3,8 +3,9 @@ import {logUiLog} from '../actions/notifications'
 
 export default function ListenLogUi () {
   engine.listenOnConnect('ListenLogUi', () => {
-    engine.listenGeneralIncomingRpc('keybase.1.logUi.log', (params: {text: Text, level: LogLevel}) => {
+    engine.listenGeneralIncomingRpc('keybase.1.logUi.log', (params: {text: Text, level: LogLevel}, response:any) => {
```

Comment:
@marco has a higher level fix for this on the electron side. Does the daeomon need to wait at all? Why not fire and forget for logging?